jersey-servers/src/main/java/com/palantir/remoting2/servers/jersey/JsonExceptionMapper.java

```diff
@@ -86,7 +74,7 @@ public final Response toResponse(T exception) {
             // simply write out the exception message
             builder = Response.status(status);
             builder.type(MediaType.TEXT_PLAIN);
-            builder.entity(exceptionMessage);
+            builder.entity(Objects.toString(exception.getMessage()));
```

Comment:
I don't think we should be propagating the exception message here as a fallback (it might not be safe). instead, we should write out the string `"Refer to the server logs with this errorId: " + errorId`